src/Mealz/MealBundle/Service/SlotService.php

```diff
@@ -70,12 +71,23 @@ public function updateState(Slot $slot, string $state): void
         $this->em->flush();
     }
 
-    public function delete(Slot $slot): void
+    public function delete(array $parameters): void
     {
-        $slot->setDeleted(true);
```

Comment:
Warum hast du hier die Logik geändert? Das Flag als deleted hat sicher einen sinn in richtung seiteneffekte.
es war schon gut so mit dem Slot object als übergabe parameter